system/HTTP/Response.php

```diff
@@ -631,7 +631,7 @@ public function setCookie(
 			{
 				if (isset($name[$item]))
 				{
-					$$item = $name[$item];
```

Comment:
This is not a typo, it called variable variables. See http://php.net/manual/en/language.variables.variable.php